deployment/network-operator/templates/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -60,6 +60,31 @@ spec:
         ]
       }
   {{- end }}
+  {{- if .Values.sriovDevicePlugin.deploy }}
+  sriovDevicePlugin:
+    # {{ required "A valid value for .Values.rdmaSharedDevicePlugin.resources is required" .Values.rdmaSharedDevicePlugin.resources }}
+    image: {{ .Values.sriovDevicePlugin.image }}
```

Comment:
I don't understand why we have the comment on the rdmaSharedDevicePlugin

Reply:
Done